nixos/modules/system/boot/systemd.nix

```diff
@@ -746,6 +746,86 @@ in
       '';
     };
 
+    systemd.tmpfiles.paths = mkOption {
+      description = ''
+        Rules for creating and cleaning up temporary files
+        automatically. See
+        <citerefentry><refentrytitle>tmpfiles.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+        for the exact format.
+      '';
```

Comment:
The description should distinguish itself from `systemd.tmpfiles.rules` - `rules` is an array and this one isn't.

Reply:
What about:
~~~
Attributes to automaticly create and remove files. See
<citerefentry><refentrytitle>tmpfiles.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>
for the exact format.
~~~